oem/ibm/libpldmresponder/oem_ibm_handler.cpp

```diff
@@ -1672,7 +1672,7 @@ void pldm::responder::oem_ibm_platform::Handler::
     processPowerCycleOffSoftGraceful()
 {
     pldm::utils::PropertyValue value =
-        "xyz.openbmc_project.State.Host.Transition.Reboot";
+        "xyz.openbmc_project.State.Host.Transition.ForceWarmReboot";
```

Comment:
although this seems like a wierd combination where we do forcewarmreboot - when the state set in pldm is soft off graceful , but approving is as phyp team wants this behavior.